docs/errors/E0427.md

```diff
@@ -0,0 +1,29 @@
+# E0427: missing 'break;' or '// fallthrough' comment between statement and 'case'
+
+Switch Cases in javascript fallthrough to the next case if the `break` statement is not added at the end of the case.
+Since there is no explicit way of communication whether the fallthrough is intentional or not, it is recommended to use a comment indicating fallthrough.
+
+```javascript
+function test (c) {
+  switch (c) {
+    case 1:
+      foo();
+    default:
+      bar();
```

Comment:
These examples assume that globals `foo` and `bar` exist, but they don't. This causes warning E0057 ('use of undeclared variable: foo').

To fix this, I suggest writing `console.log` calls instead. (The `console` global should exist in these examples.)